examples/playground/components/after-paint/with-ssr.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 
-import { controlFetch, isServer } from '../../utils';
+import { createSuspendableData } from '../../utils';
```

Comment:
These examples changes are just quality-of-life changes, making them more strict in behaviour and fully StrictMode compatible